src/api/AuthServer/AuthServer/Configuration/JwtSettings.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace AuthServer.Configuration
+{
+    public class JwtSettings
+    {
+        public string SecretKey { get; set; }
+        public string Issuer { get; set; }
+        public string Audience { get; set; }
```

Comment:
At the development/deployment stage, we cannot know the audience. Not needed.
Empty spaces between property declarations needed